package.json

```diff
@@ -39,8 +39,8 @@
     "yup": "^0.32.11"
   },
   "scripts": {
-    "build": "esbuild app/javascript/*.* --bundle --sourcemap --outdir=app/assets/builds --loader:.png=dataurl",
-    "build:css": "sass ./app/assets/stylesheets/application.bootstrap.scss ./app/assets/builds/application.css --no-source-map --load-path=node_modules"
+    "build": "esbuild app/javascript/*.* --bundle --sourcemap --outdir=app/assets/builds --loader:.png=dataurl --minify",
```

Comment:
The `--minify` will cause esbuild to make a complete minification to the js bundle(s) and furthermore set the `process.env.NODE_END` to `production` which what we need on a production environment.
However this will cause the following issues for development:
1. Dev js assets will be minified (no practical debugging is possible).
2. Any js package that is conform with the standard will then run its production version which will not usually include any verbose errors or logging.

Reply:
Let me see if there's a way to run 2 different scripts based on env variable